src/Hangfire.DynamicJobs/DynamicJobRecurringJobManagerExtensions.cs

```diff
@@ -138,11 +139,28 @@ private static Job ToDynamicJob([NotNull] Job job, [CanBeNull] IEnumerable<JobFi
             if (job == null) throw new ArgumentNullException(nameof(job));
 
             var invocationData = InvocationData.SerializeJob(job);
+
+            //get display name form original job
+            var jobDisplayNameAttribute = job.Method.GetCustomAttributes<JobDisplayNameAttribute>(inherit: true).FirstOrDefault();
+            //TODO: maybe is better resolve name here and serialize as string?
+
+            //get queue filter from original job            
+            ////    Job.Method.GetMethodFilterAttributes() inaccessibile
+            ////    ReflectedAttributeCache.GetMethodFilterAttributes(job.Method); inaccessible
+            var typeFiltes = job.Type.GetTypeInfo().GetCustomAttributes(typeof(QueueAttribute), inherit: true).Cast<QueueAttribute>();
+            var methodFiltes = job.Method.GetCustomAttributes(typeof(QueueAttribute), inherit: true).Cast<QueueAttribute>();
+
+            //generate list of filters for dynamicjob
+            var dynamicfilters = new List<JobFilterAttribute>(filters?.ToArray() ?? Enumerable.Empty<JobFilterAttribute>());
```

Comment:
I think we should concentrate on description only in this pull request. Automatic filter attribute acquisition is great, but it has some gotchas – since arguments (or their types) of a target method can be unavailable at runtime, this solution will have a lot of limitations. Currently `Hangfire.DynamicJobs` extension requires all the filters to be passed explicitly, and this is the only obvious solution I see without hidden problems.